src/strategies.js

```diff
@@ -1,15 +1,16 @@
-import initiationStrategies from './initiationStrategies';
-
-export const strategies = initiationStrategies({
-  nextTick(pluginFn, $$) {
-    return window.setTimeout(() => pluginFn(), 0);
-  },
-
-  immediate(pluginFn, $$) {
-    return pluginFn() || {};
-  },
-
-  hover(pluginFn, $$) {
-    return $$.one('mouseover', () => pluginFn());
-  }
-});
+export function nextTick(pluginFn, _el) {
+  return window.setTimeout(() => pluginFn(), 0);
+}
+export function immediate(pluginFn, _el) {
+  return pluginFn() || {};
+}
+export function hover(pluginFn, el) {
+  const isJqueryEl = el.jquery !== undefined;
+  return isJqueryEl ?
+    el.one('mouseover', pluginFn) :
+    el.addEventListener('mouseover', pluginFn, { once: true });
+}
+export function fallback(strategy, fallback) {
+  console.warn(`Strategy ${strategy} not found, falling back to ${fallback.name} strategy.`);
```

Comment:
Maybe just `throw InitiationNotSupportedError`? This is a major mistake, it doesn't feel like a warning is sufficient.

Reply:
So then not fallback at all.

Reply:
Yeah, that's what I prefer anyway. I don't think throwing a "function undefined" method is helpful, but having a strict interface is good too.

Reply:
Actually I'm wrong, the fallback with warning is a good idea.

Using a widget into a different environment probably shouldn't break it.

Reply:
> Actually I'm wrong, the fallback with warning is a good idea.
> 
> Moving a widget into a different environment probably shouldn't break it.

Yeah, that was my line too 👍